src/main/java/duke/command/Ui.java

```diff
@@ -0,0 +1,110 @@
+package duke.command;
+
+import duke.exception.DukeException;
+import duke.task.Task;
+
+import java.util.Scanner;
+
+/**
+ * Manages input and output
+ */
+public class Ui {
+    private static TaskList taskList;
```

Comment:
I understand you introduced a TaskList variable here so that the Ui can handle printing of commands like AddCommand and DeleteCommand but I personally didn't do it as I was afraid of introducing unnecessary dependencies. What do you think of printing output of commands through the .execute() method in commands themselves, or maybe introducing a method in the Ui class that takes in a command as a parameter?